h2o-py/h2o/targetencoder.py

```diff
@@ -18,7 +18,8 @@
 class TargetEncoder(object):
 
     """
-    Deprecated API. Please use H2OTargetencoderEstimator instead.
+    :deprecated:
+    Use :func:`h2o.estimators.targetencoder.H2OTargetEncoderEstimator` instead.
```

Comment:
proper documentation for deprecation in Python, generating link to the replacement class.